File: backend/app/schemas/paper.py
from datetime import datetime
from pydantic import BaseModel, ConfigDict


class PaperResponse(BaseModel):

    model_config = ConfigDict(from_attributes=True)

    id: str
    project_id: str

    filename: str
    original_filename: str

    title: str | None

    page_count: int

    status: str

    created_at: datetime

File: Client/paper-mind/src/types/paper.ts
export interface Paper {
  updated_at: string | number | Date;
  authors: any;
  id: string;
  project_id: string;

  filename: string;
  original_filename: string;

  title: string | null;

  page_count: number;

  status: string;

  created_at: string;
}